product_search/product_search.py

```diff
@@ -0,0 +1,46 @@
+"""Program that returns inventory details based for an inventory item"""
+
+import sys
+from data_management import access_json_data
+
+data = access_json_data(sys.argv[1])
+
+
+def get_product_name():
+    """Get a name of a product from the user"""
+    return input("What is the product name? ").title()
+
+
+def item_inventory_details(item_name):
```

Comment:
So what I would do is pass in the dictionary of inventory to this function. This would necessitate your `main` function getting the filename (which it's already doing) and parsing the values from the file into a variable that you can then use. So in a nutshell all that functionality should live one level up. So Line 6 in this file should live in `main`.

Reply:
@paulghaddad I'm not clear on this comment. Let's discuss in person.